### PR TITLE
DOCS-309 adding redirect uri SmartApp setting documentation

### PR DESCRIPTION
**Pull request by unixbeast**

### CODE REVIEW

File: smartapp-web-services-developers-guide/authorization.rst [hunk 1]
```diff
@@ -51,7 +51,7 @@ parameter      value
 response_type  Use ``code`` to obtain the authorization code.
 client_id      The OAuth client ID of the SmartApp.
 scope          This should always be "app" for this authorization flow.
-redirect_uri   The URI of your server that will receive the authorization code.
+redirect_uri   The URI of your server that will receive the authorization code. This uri must match one of the redirect URIs specified in the SmartApp settings, otherwise validation will fail.
```
Comment (jimmyjames) [inline smartapp-web-services-developers-guide/authorization.rst]: Inconsistent use of capitalization for uri. Probably stick with capitalization (URI) unless it is a code literal or parameter name.

File: smartapp-web-services-developers-guide/smartapp.rst [hunk 2]
```diff
@@ -18,6 +18,26 @@ In the *OAuth* section, check the box to enable OAuth.
 A client ID and secret will be generated for this SmartApp.
 These will be used as part of the OAuth flow to obtain an access token for this SmartApp.
 
+There is an option to specify a Redirect URI.
+This URI will be used to validate the ``redirect_uri`` passed in with the request for the authorization code.
+The value of this field can be a single value, or a comma delimited list of values.
```
Comment (jimmyjames) [inline smartapp-web-services-developers-guide/smartapp.rst]: comma-delimited is hyphenated.

**Comment by jimmyjames** [comment]
What is the impact to existing installed web services SmartApps? Are there steps they need to take? If so, we should document them.

**Comment by unixbeast** [comment]
There are no actions necessary as this is an optional setting.

**Comment by jimmyjames** [comment]
Cool. Aside from some nit-picky grammar things, looks good. @llinder should give a thumbs-up for content though as the SME.

**Comment by unixbeast** [comment]
@llinder can you take a quick look at this pull request before we merge into the docs? thanks.

Dave

**Comment by llinder** [comment]
👍

**Pull request merged**